src/main/java/org/fundacionjala/sfdc/pageobjects/components/TopMenuLightning.java

```diff
@@ -0,0 +1,13 @@
+package org.fundacionjala.sfdc.pageobjects.components;
+
+public class TopMenuLightning extends TopMenu{
+    @Override
+    public void switchTheme() {
+
+    }
+
+    @Override
+    public void waitUntilPageObjectIsLoaded() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)